bitnami/mongodb/values.yaml

```
@@ -179,6 +179,9 @@ tls:
   ## @param tls.image.tag Init container TLS certs setup image tag (immutable tags are recommended)
   ## @param tls.image.pullPolicy Init container TLS certs setup image pull policy
   ##
+  extraDnsNames: []
+  ## Add extra dns names to the CA, can solve x509 auth issue for pod clients
+  ##
```

It's necessary to add the metadata to generate de tables of the README.md

```suggestion
  extraDnsNames: []
  ## @param tls.extraDnsNames Add extra dns names to the CA, can solve x509 auth issue for pod clients
  ##
```

---

Ho ok. never used that before :)